runtime/compiler/control/J9Options.cpp

```diff
@@ -1235,6 +1237,15 @@ static bool JITServerParseCommonOptions(J9JavaVM *vm, TR::CompilationInfo *compI
       TR::Options::setVerboseOption(TR_VerboseJITServerConns);
       }
 
+   if (xxJITServerAOTmxArgIndex >= 0)
+      {
+      uint32_t aotMaxBytes = 0;
+      if (GET_INTEGER_VALUE(xxJITServerAOTmxArgIndex, xxJITServerAOTmxOption, aotMaxBytes) == OPTION_OK)
```

Comment:
This is still using `GET_INTEGER_VALUE` instead of `GET_MEMORY_VALUE`, which requires specifying the limit in bytes if I understand correctly.